### PR TITLE
1.x - update submit work and contract.myMethod.send docs

### PR DESCRIPTION
**Pull request by luu-alex**
## Description
#5200
Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).
https://web3js.readthedocs.io/en/v1.7.4/web3-eth-contract.html#methods-mymethod-send
`myContract.methods.myMethod([param1[, param2[, ...]]]).send(options[, callback])`

should be `myContract.methods.myMethod(param1, param2,...).send(options[, callback])`

and [submitwork](https://web3js.readthedocs.io/en/v1.7.4/web3-eth.html#id114) should not take in a list but

should be `web3.eth.submitWork(
    "0x0000000000000001",
    "0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef",
    "0xD1FE5700000000000000000000000000D1FE5700000000000000000000000000"
)`

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

**Review by jdevcs — APPROVED** [review]

**Review by avkos — APPROVED** [review]

**Review by alokeutpal — APPROVED** [review]

**Review by Muhammad-Altabba — APPROVED** [review]

File: docs/web3-eth.rst [hunk 1]
```diff
@@ -2015,11 +2015,11 @@ Example
 
 .. code-block:: javascript
 
-    web3.eth.submitWork([
+    web3.eth.submitWork(
```
Comment (Muhammad-Altabba) [inline docs/web3-eth.rst:2018]: It seems the method should accept an array of strings:
https://github.com/ChainSafe/web3.js/blob/8d369a9e76267df9ce666283521c1fd6d3f77fc4/packages/web3-eth/types/index.d.ts#L388
```js
    submitWork(
        data: [string, string, string],
        callback?: (error: Error, result: boolean) => void
    ): Promise<boolean>;
```


And here are also usage:

- https://vscode.dev/github/ChainSafe/web3.js/blob/SriCharanGundla-fix-typo/packages/web3-eth/types/tests/eth.tests.ts#L583
- https://vscode.dev/github/ChainSafe/web3.js/blob/SriCharanGundla-fix-typo/packages/web3-eth/types/tests/eth.tests.ts#L590

Did you miss updating related stuff? Or do I miss something here?

Comment (jdevcs) [reply docs/web3-eth.rst:2018]: @Muhammad-Altabba submit work expects 3 params: 
https://github.com/web3/web3.js/blob/47b9769d99c70ea985a5e639c9ca37ebb5c7cde9/packages/web3-eth/src/index.js#L564

passing array gives:
```
/node_modules/web3-core-method/lib/index.js:113
        throw errors.InvalidNumberOfParams(args.length, this.params, this.name);
        ^

Error: Invalid number of parameters for "submitWork". Got 1 expected 3!

```
In types file link you shared , types are not updated. cc: @luu-alex these also needs update.

Comment (luu-alex) [reply docs/web3-eth.rst:2018]: You seem to be correct that the docs and usage takes in array but only passing in the 3 seperate params work. i will update the testcases. thank you! updated

**Review by jdevcs — COMMENTED** [review]
failing CI tests in this PR should be fixed under : https://github.com/web3/web3.js/issues/6242

**Pull request merged**